### PR TITLE
Got build working again

### PR DESCRIPTION
**Pull request by cyian-1756**
I got the build working on Go 1.11.2 by removing 2 bees (tinderbee and slackbee) and making some minor changes so how error pages are called

The tinderbee was removed because the lib it used has been removed from github

### CODE REVIEW

**Comment by rubiojr** [comment]
@cyian-1756 building fine here with:

```
[0][rubiojr@mantaray] go version
go version go1.12.1 darwin/amd64
```

What kind of error are you getting?

**Comment by cyian-1756** [comment]
> What kind of error are you getting?

I was getting a 404 error on the tinderbee as the tinder lib it relied on has been pulled from github

The slackbee was throwing an error because the lib it uses has made breaking api changes (The exact error was about a wrong number of arguments being passed to postMessage)

The 2 other error messages I got were the same as in #212

**Comment by rubiojr** [comment]
@cyian-1756 ah I see, it seems the way we vendor things changed in https://github.com/muesli/beehive/pull/185 where we stopped using muesli/beehive-vendor, so things will break if 3rd party libs disappear from the interwebs.

Perhaps @penguwin is interested in fixing the Tinder bee (per https://github.com/muesli/beehive/pull/160), instead of removing it, assuming that can be done. In any case, we should remove it from `Gopkg.toml` if we're 🔥 it.

You'll want to bump the slack dep in https://github.com/muesli/beehive/blob/master/Gopkg.toml#L138 to 0.5.0, otherwise the build will fail if we pull 0.2.

I'm not sure about the changes to `bees_post.go` and `chains_post.go`, but it seems there's a legit issue if you don't build against smolder `e6752d792454819bc63dec1a7a6f04bd7d539c7c` (current HEAD), so you'll want set that in `Gopkg.toml`, having master there is not enough:

```
api/resources/bees/bees_post.go:63:31: too many arguments in call to smolder.ErrorResponseHandler
	have (*restful.Request, *restful.Response, error, *smolder.ErrorResponse)
	want (*restful.Request, *restful.Response, *smolder.ErrorResponse)
api/resources/bees/bees_post.go:63:80: not enough arguments in call to smolder.NewErrorResponse
	have (number, error, string)
	want (int, bool, interface {}, string)
# github.com/muesli/beehive/api/resources/chains
api/resources/chains/chains_post.go:63:31: too many arguments in call to smolder.ErrorResponseHandler
	have (*restful.Request, *restful.Response, nil, *smolder.ErrorResponse)
	want (*restful.Request, *restful.Response, *smolder.ErrorResponse)
api/resources/chains/chains_post.go:63:80: not enough arguments in call to smolder.NewErrorResponse
	have (number, nil, string)
	want (int, bool, interface {}, string)
make: *** [build] Error 2
``` 

@muesli should probably have a closer look at the smolder changes, I'm not overly familiar with it.

**Comment by prologic** [comment]
I really don't see the point of the TinderBee. It should just be removed and beehive resurrected as a useful alternative to the various SaaS based IFTT-type serivces.

File: bees/slackbee/slackbee.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (mod *SlackBee) Action(action bees.Action) []bees.Placeholder {
 		msgParams := slack.NewPostMessageParameters()
 		msgParams.LinkNames = 1
 		for _, to := range tos {
-			_, _, err := mod.client.PostMessage(to, text, msgParams)
+			_, _, err := mod.client.PostMessage(to, slack.MsgOptionText(text, false))
```
Comment (rubiojr) [inline bees/slackbee/slackbee.go]: A better fix would be to set `msgParams := slack.PostMessageParameters{}`, then `_, _, err := mod.client.PostMessage(to, slack.MsgOptionPostMessageParameters(msgParams))`.

That shouldn't change the behavior.

File: bees/slackbee/slackbee.go [hunk 2]
```diff
@@ -63,10 +63,10 @@ func (mod *SlackBee) Action(action bees.Action) []bees.Placeholder {
 			}
 		}
 
-		msgParams := slack.NewPostMessageParameters()
+		msgParams := slack.PostMessageParameters{}
 		msgParams.LinkNames = 1
 		for _, to := range tos {
-			_, _, err := mod.client.PostMessage(to, slack.MsgOptionText(text, false), slack.MsgOptionPostMessageParameters(msgParams))
+			_, _, err := mod.client.PostMessage(to, text, msgParams)
```
Comment (rubiojr) [inline bees/slackbee/slackbee.go]: @cyian-1756 missed something in my last comment, this should be:

```go
_, _, err := mod.client.PostMessage(to, slack.MsgOptionText(text, false), slack.MsgOptionPostMessageParameters(msgParams))
```

Comment (rubiojr) [reply bees/slackbee/slackbee.go]: Ah nevermind, I see you've addressed this in a different commit. Thanks!

**Comment by muesli** [comment]
We could use the tinder library from beehive-vendor and push that to a new repository on GitHub. I'm not sure it's worth the effort though. An unmaintained library will break sooner than later and I don't think anyone here wants to really maintain a tinder library themselves.

**Comment by muesli** [comment]
I appreciate all the effort you put into this! I'll probably squash and merge this tonight, but I also think at this point we should just move on to using go modules. Let me figure out in which order I want to approach things :laughing:

**Pull request merged**